pl-faded-parsons.js

```diff
@@ -577,10 +607,16 @@ class ParsonsWidget {
   }
   codelineSummary(line, idx) {
     // this schema is used in pl-faded-parsons.py `read_lines`!
```

Comment:
another small thing, could we keep this comment with the return statement so it warns against anyone changing it?